### PR TITLE
Buffs the Battlemage Armor, deletes Battlemage Armor charges

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Greatly improves Battlemage Armor, giving it Syndicate Hardsuit grade armor with a recharging shield, along with spaceproofing. 

EDIT: At the behest of the voices in my head and on the balance team, the shields no longer recharge and return to 15 charges. Armor and spaceproofing remain.

Also buffs Archmage Armor with an extremely rapid recharging shield for admeme use.

Removes the armor charges since they're still worthless and will never return or be loved.

## Why It's Good For The Game
Battlemage armor sucks. You get exactly 15 hits before your armor is now utterly worthless as its base armor values are terrible.

The charges to replenish this are also a complete scam and not worth it in the slightest. The suit not even being spaceproof is insult to injury.

Now, it should be a proper set of wizard power armor that makes wizards a good bit less squishy.

## Testing
Loaded in, donned battlemage armor. Jaunted to space, was fine. Let turrets shoot me before jaunting away, shield broke after 15 hits and never recharges.

## Changelog
:cl:
tweak: Buffed Battlemage Armor, increasing armor values and adding spaceproofing.
del: Deleted Battlemage Armor charges
/:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
Here too, changelog could use more details

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -475,18 +475,11 @@
 //Weapons and Armors
 /datum/spellbook_entry/item/battlemage
 	name = "Battlemage Armour"
-	desc = "An ensorceled suit of armour, protected by a powerful shield. The shield can completely negate sixteen attacks before being permanently depleted. Despite appearance it is NOT spaceproof."
+	desc = "An ensorceled suit of protective yet light armour, protected by a powerful shield. The shield can completely negate three attacks at a time before needing to recharge. The armor is also spaceproof."
```
[the remark this answers is not in the document]
Comment (Contrabang) [reply code/game/gamemodes/wizard/spellbook.dm:478]: Given that the actual item uses `armor` it should probably be the american version

[the remark this answers is not in the document]
Comment (PopeDaveThe3th) [reply code/game/gamemodes/wizard/spellbook.dm]: Switched all to American spelling.

**Comment by PopeDaveThe3th** [comment]
> Here too, changelog could use more details

Done.

File: code/game/gamemodes/wizard/spellbook.dm [hunk 2]
```diff
@@ -474,19 +474,12 @@
 
 //Weapons and Armors
 /datum/spellbook_entry/item/battlemage
-	name = "Battlemage Armour"
-	desc = "An ensorceled suit of armour, protected by a powerful shield. The shield can completely negate sixteen attacks before being permanently depleted. Despite appearance it is NOT spaceproof."
+	name = "Battlemage Armor"
+	desc = "An ensorceled suit of protective yet light armor, protected by a powerful shield. The shield can completely negate three attacks at a time before needing to recharge. The armor is also spaceproof."
```
Comment (DGamerL) [inline code/game/gamemodes/wizard/spellbook.dm:478]:
```suggestion
	desc = "An ensorceled spaceproof suit of protective yet light armor, protected by a powerful shield. The shield can completely negate three attacks before needing to recharge."
```

File: code/modules/clothing/suits/wiz_robe.dm [hunk 3]
```diff
@@ -186,25 +186,27 @@
 
 /obj/item/clothing/suit/space/hardsuit/shielded/wizard
 	name = "battlemage armor"
-	desc = "Not all wizards are afraid of getting up close and personal. Not spaceproof despite its appearance."
+	desc = "Not all wizards are afraid of getting up close and personal. It is spaceproof."
```
Comment (DGamerL) [inline code/modules/clothing/suits/wiz_robe.dm:189]:
```suggestion
	desc = "Not all wizards are afraid of getting up close and personal."
```

File: code/modules/clothing/suits/wiz_robe.dm [hunk 4]
```diff
@@ -186,25 +186,27 @@
...
 	icon_state = "hardsuit-wiz"
 	item_state = "wiz_hardsuit"
-	recharge_rate = 0
-	current_charges = 15
-	recharge_cooldown = INFINITY
+	recharge_rate = 1
+	current_charges = 3
+	max_charges = 3
+	recharge_cooldown = 50
+	recharge_delay = 100
```
Comment (DGamerL) [inline code/modules/clothing/suits/wiz_robe.dm:196]: Are these in ticks? If so, could you use the define instead?

File: code/modules/clothing/suits/wiz_robe.dm [hunk 5]
```diff
@@ -186,25 +186,27 @@
...
-	armor = list(MELEE = 20, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 10, RAD = 10, FIRE = INFINITY, ACID = INFINITY)
+	armor = list(MELEE = 35, BULLET = 50, LASER = 20, ENERGY = 10, BOMB = 25, RAD = 50, FIRE = INFINITY, ACID = INFINITY)
 	slowdown = 0
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	magical = TRUE
 
 /obj/item/clothing/suit/space/hardsuit/shielded/wizard/arch
 	desc = "For the arch wizard in need of additional protection."
-	recharge_rate = 1
+	recharge_rate = 5
```
Comment (DGamerL) [inline code/modules/clothing/suits/wiz_robe.dm:209]: Same here

**Review by DGamerL — COMMENTED** [review]
Some nitpicks

**Review by DGamerL — APPROVED** [review]
LGTM, nice work keeping up with the reviews :)

**Comment by matttheficus** [comment]
This might need an increase to 3 cost if its an Infinite Shielded Robe. This makes killing wizards a LOT more difficult if they just jaunt away every time they lose the shield.

I do like the item not being dogass useless though.

**Review by S34NW — APPROVED** [review]

**Pull request merged**